DataFormats/Detectors/TRD/include/DataFormatsTRD/RawData.h

```diff
@@ -214,10 +214,168 @@ struct TRDFeeID {
       uint8_t side : 1;        // the A=0 or C=1 side of the supermodule being readout
       uint8_t unused1 : 3;     // seperate so easier to read in hex dumps
       uint8_t supermodule : 8; // the supermodule being read out 0-17
-    } __attribute__((__packed__));
+    }__attribute__((__packed__));
   };
 };
 
+
+/// \structure DigitHCHeader 
+/// \brief Digit version of the TrackletHCHeader above, although contents are rather different.
+//  TODO come back and comment the fields or make the name more expressive, and fill in the jjjjjjj
+struct DigitHCHeader{
+//
+  //             10987654321098765432109876543210
+  // uint32_t:   00000000000000000000000000000000
+  //             1zzzz  pppppppp        pppppppp1
+  union {//page 109 section 15.6.1 in tdp
+    uint32_t word0;
+    struct {
+        uint32_t res0 : 2;   
+        uint32_t side : 1;
+        uint32_t stack : 3;
+        uint32_t layer : 3;
+        uint32_t supermodule : 5;
+        uint32_t addHCW: 3;
+        uint32_t minor : 7;
+        uint32_t major : 7;
+        uint32_t version : 1;
+    }__attribute__((__packed__));
+  };
+
+  //             10987654321098765432109876543210
+  // uint32_t:   00000000000000000000000000000000
+  //             1zzzz  pppppppp        pppppppp1
+  union {//page 109 section 15.6.2 in tdp
+    uint32_t word1;
+    struct {
+        uint32_t res1 : 2;    
+        uint32_t ptrigcount : 4; 
+        uint32_t ptrigphase : 4; 
+        uint32_t bunchcrossing : 16; 
+        uint32_t numtimebins : 6;   
+    }__attribute__((__packed__));
+  };
+  //             10987654321098765432109876543210
+  // uint32_t:   00000000000000000000000000000000
+  union { //page 109 section 15.6.3 in tdp
```

Comment:
Usually, we only have one additional header word in the data. Not sure if this will create an issue if you don't read them.